py-polars/polars/dataframe/frame.py

```diff
@@ -4641,6 +4641,16 @@ def groupby(
             Settings this to ``True`` blocks the possibility
             to run on the streaming engine.
 
+            .. note::
+
+                Within each group, the order of rows is always preserved, regardless
+                of this argument.
```

Comment:
It's a bit more readable without the extra blank line, in my opinion.

```suggestion
            .. note::
                Within each group, the order of rows is always preserved, regardless
                of this argument.
```

Reply:
sure, thanks! I don't think it makes a difference when the docs are rendered, but still, good to have one less line of code

Reply:
Exactly, thanks!